src/components/AddPost.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+const AddPost = ({ onAdd }) => {
+  const [userId, setUserId] = useState(1);
+  const [message, setMessage] = useState("");
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+
+    if (!message) {
+      alert("Please add post");
+      return;
+    }
+
+    onAdd({ userId, message });
+
+    setUserId(1);
```

Comment:
we dont need this state

Reply:
add this id manually to the object, later will get userId from SecurityContext on server